code/data_structures/stack/balanced_expression/balanced_expression.cpp

```diff
@@ -0,0 +1,51 @@
+#include<iostream>
+using namespace std;
+
```

Comment:
Don't forget to add the required comment as stated in the [contribution guidelines](https://github.com/OpenGenus/cosmos/wiki/contribute).